#
#Header header
#The Joints of the Aibo
float64 jointLF1
float64 jointLF2     
float64 jointLF3     
float64 jointLH1     
float64 jointLH2 
float64 jointLH3 
float64 jointRF1
float64 jointRF2
float64 jointRF3
float64 jointRH1
float64 jointRH2
float64 jointRH3
float64 tailPan
float64 tailTilt
float64 headTilt
float64 headPan
float64 headNeck
float64 mouth
